src/app/course/_forms/course-event.form.ts

```diff
@@ -53,7 +53,7 @@ export class CourseEventForm {
     }
 
     /**
-     * Creates a FormGroup for a Course Event with existing data.
+     * Creates a FormGroup for a Course Assessment with existing data.
```

Comment:
here Course Event refers to out model CourseEvent, therefore should not be changed.